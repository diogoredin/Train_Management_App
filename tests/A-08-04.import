SERVICE|180|51.5|05:47|Porto - Campanhã|08:10|Casa Branca|11:23|Faro
SERVICE|694|12.2|16:57|Évora|17:08|Casa Branca|18:36|Lisboa - Oriente
SERVICE|420|11.45|08:36|Valenca|09:09|Viana do Castelo|09:50|Nine|10:18|Porto - Campanhã
SERVICE|5500|11.35|15:48|Elvas|18:25|Entroncamento
PASSENGER|Obi-Wan
PASSENGER|Yoda
ITINERARY|0|2017-10-18|180/Porto - Campanhã/Faro
ITINERARY|0|2017-09-19|180/Porto - Campanhã/Faro
ITINERARY|0|2017-10-17|180/Porto - Campanhã/Faro
ITINERARY|0|2017-10-14|180/Porto - Campanhã/Casa Branca|694/Casa Branca/Lisboa - Oriente
ITINERARY|0|2017-10-15|180/Porto - Campanhã/Casa Branca|694/Casa Branca/Lisboa - Oriente
ITINERARY|0|2017-10-19|180/Porto - Campanhã/Casa Branca|694/Casa Branca/Lisboa - Oriente
ITINERARY|1|2017-11-18|180/Porto - Campanhã/Casa Branca
