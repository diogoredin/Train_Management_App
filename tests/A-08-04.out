Menu principal
1 - Reiniciar
2 - Abrir
3 - Guardar
4 - Consulta de serviços
5 - Gestão de passageiros
6 - Gestão de itinerários
0 - Sair
Escolha uma opção: Menu de gestão de itinerários
1 - Mostrar todos os itinerários
2 - Mostrar itinerários associados a um passageiro
3 - Registar itinerário para um passageiro
0 - Sair
Escolha uma opção: Identificador do passageiro: == Passageiro 0: Obi-Wan ==

Itinerário 1 para 2017-09-19 @ 51.50
Serviço #180 @ 51.50
05:47 Porto - Campanhã
08:10 Casa Branca
11:23 Faro

Itinerário 2 para 2017-10-14 @ 32.76
Serviço #180 @ 21.92
05:47 Porto - Campanhã
08:10 Casa Branca
Serviço #694 @ 10.84
17:08 Casa Branca
18:36 Lisboa - Oriente

Itinerário 3 para 2017-10-15 @ 32.76
Serviço #180 @ 21.92
05:47 Porto - Campanhã
08:10 Casa Branca
Serviço #694 @ 10.84
17:08 Casa Branca
18:36 Lisboa - Oriente

Itinerário 4 para 2017-10-17 @ 51.50
Serviço #180 @ 51.50
05:47 Porto - Campanhã
08:10 Casa Branca
11:23 Faro

Itinerário 5 para 2017-10-18 @ 51.50
Serviço #180 @ 51.50
05:47 Porto - Campanhã
08:10 Casa Branca
11:23 Faro

Itinerário 6 para 2017-10-19 @ 32.76
Serviço #180 @ 21.92
05:47 Porto - Campanhã
08:10 Casa Branca
Serviço #694 @ 10.84
17:08 Casa Branca
18:36 Lisboa - Oriente
Menu de gestão de itinerários
1 - Mostrar todos os itinerários
2 - Mostrar itinerários associados a um passageiro
3 - Registar itinerário para um passageiro
0 - Sair
Escolha uma opção: Menu principal
1 - Reiniciar
2 - Abrir
3 - Guardar
4 - Consulta de serviços
5 - Gestão de passageiros
6 - Gestão de itinerários
0 - Sair
Escolha uma opção: 